testutils/utils.go

```diff
@@ -0,0 +1,25 @@
+package testutils
+
+import (
+	"testing"
+
+	cometdb "github.com/cometbft/cometbft-db"
+	dbm "github.com/cosmos/cosmos-db"
+	"github.com/stretchr/testify/require"
+)
+
+func NewCometMemDB(t *testing.T) cometdb.DB {
+	db := cometdb.NewMemDB()
+	t.Cleanup(func() {
+		require.NoError(t, db.Close())
+	})
+	return db
+}
```

Comment:
Again unsure about why this was introduced.

Reply:
Good point. Answered above.